No_longer_slaves.tex

```diff
@@ -0,0 +1,50 @@
+\documentclass{leadsheet}
+
+\begin{document}
+
+\begin{song}[transpose={0}]{title={No longer Slaves},composer={Brian Johnson, Joel Case, Jonathan David Helser},key={A},tempo={74}}
+
+\begin{schedule}
+I -- S1 -- R -- ZS -- S2 -- R -- R -- ZS -- B -- R
+\end{schedule}
+
+\begin{intro}
+||: ^{F#m}\_\_ ^{E}\_\_ | ^{A}\_\_ ^{D}\_\_ :||
+\end{intro}
+
+\begin{verse}
+You unrav- | ^{A}el me, with a melo- | dy 
+You sur- | ^{D}round me ^Ewith a | ^Asong \\
+Of deliv- | ^{A}erance, from my ene- | mies 
+Till | ^{D}all my ^Efears are | ^Agone
+\end{verse}
+
+\begin{chorus}
+I’m no | ^Dlonger a ^Eslave to | ^Afear \_ |\\
+^{F#m}I~am a ^Echild of | ^AGod \\
+I’m no | ^Dlonger a ^Eslave to | ^Afear \_ | \\
+\textsuperscript{1st Ending}~^{F#m}I~am a ^Echild of | ^{F#m}God \\
+\textsuperscript{2nd Ending}~^{F#m}I~am a ^Echild of | ^{A}God \textsuperscript{Chorus/Ending}\\
+\end{chorus}
+
+\begin{interlude}
+||: ^{F#m}\_\_ ^E\_\_ | ^{A}\_\_ ^D\_\_ :||
+\end{interlude}
+
+\begin{verse}             
+From my Moth- | ^{A}ers womb You have | ^{C#m}chosen me 
+| ^DLove has ^Ecalled my | ^Aname \\
+I’ve been |^Aborn again, into your | ^{C#m}family 
```

Comment:
Should be:
I've been born a- | ^Again, into your fami- | ^{C#m}ly;

Reply:
Has been fixed in new commit